purchase_work_acceptance_evaluation/models/work_acceptance.py

```diff
@@ -0,0 +1,60 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class WorkAcceptance(models.Model) :
+    _inherit = "work.acceptance"
+    _description = "Work Acceptance Evaluation Tab"
+
+    evaluation_result_ids = fields.One2many(
+        comodel_name="work.acceptance.evaluation.result",
+        inverse_name="wa_id",
+        string="Evaluation Results",
+        default=lambda self: self._default_evaluation_result_ids(),
+    )
+
+    def _default_evaluation_result_ids(self):
+        eval_result = self.env["work.acceptance.evaluation"].search([])
+        result = [(0, 0, {"case_id": rec.id}) for rec in eval_result]
+        return result
+
+    @api.constrains('state', 'evaluation_result_ids')
+    def _check_evaluation(self):
```

Comment:
This function can be bypassed, if `group_enable_eval_on_wa` in setting is not enable (or has_group() is false)